internal/commands/ecosystems/enrich.go

```diff
@@ -1,15 +1,14 @@
 package ecosystems
 
 import (
-	"bytes"
 	"os"
 
-	cdx "github.com/CycloneDX/cyclonedx-go"
```

Comment:
This command is now "format agnostic", allowing for input of different SBOM standards.